cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/query/NumericQueryTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright © 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.sync.internal.query;
+
+import com.cloudant.sync.documentstore.DocumentBodyFactory;
+import com.cloudant.sync.documentstore.DocumentRevision;
+import com.cloudant.sync.query.FieldSort;
+import com.cloudant.sync.query.QueryException;
+import com.cloudant.sync.query.QueryResult;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class NumericQueryTest extends AbstractQueryTestBase {
+
+    int defaultInt = 42;
+    float defaultFloat = 1.01f;
+    double defaultDouble = 1.01d;
```

Comment:
I'd prefer these three default values to be `final` so it's clearer they don't change and I think they should probably be `private` and `static` too.